README.md

```diff
@@ -1,2 +1,42 @@
 # MEICO
-Static webpage for the Mozilla Internet Ecosystem (MIECO)
+Webpage for the Mozilla Innovations team at https://future.mozilla.org and the Mozilla Internet Ecosystem (MIECO).
+
+## Getting Started
+
+This project is a Static webpage with HTML, CSS and JS. We use [ParcelJS](https://parceljs.org/) to bundle everything to the `dist/` folder. This project also uses [Jinja Templates](https://jinja.palletsprojects.com/en/3.1.x/) to enable the use of templating in this project. To convert jinja templates to HTML, we use a python script,
+
+## Installation
+These instruction assume you have Python, pip and NodeJS installed.
+
+To build MIECO from source and run the site locally, you can
+clone the repo from GitHub:
+
+```
+git clone https://github.com/mozmeao/MIECO.git
+npm install
+```
+
+Running `npm install` will install the NPM dependencies
+
+then install the python dependencies:
+
+```
+pip install -r requirements.txt
+```
+
+Once you install all of the dependencies you can run this command to view the site at http://localhost:8000/mieco :
```

Comment:
```suggestion
Once you install all of the dependencies you can run this command to view the site at http://localhost:8000/mieco:
```

Reply:
Also, how come the main URL is has `/mieco`? why isn't it just `localhost:8000`?

Reply:
It's due to the fact that the innovation team wants to eventually do something different with future.mozilla.org and wanted all mieco things to be under /mieco